nautobot/core/templates/components/panel/body_content_text.html

```diff
@@ -1,10 +1,14 @@
 {% load helpers %}
-{%  if render_as == render_options.json %}
+{%  if render_as == 'json' %}
     {{ text_content | render_json }}
-{% elif render_as == render_options.yaml %}
-    {{ text_content | render_yaml }}
-{% elif render_as == render_options.markdown %}
+{% elif render_as == 'json_pretty' %}
+    {{ text_content | render_json_pretty }}
+{% elif render_as == 'yaml' %}
+    <pre>{{ text_content | render_yaml }}</pre>
```

Comment:
This is the place where I started thinking about consistency between those two formats. Whether to keep `render_json_pretty` and add `render_yaml_pretty` or keep it simple and just add another filter to do this like that:
```
{% elif render_as == 'json_pretty' %}
    {{ text_content | render_json | pretty }}
{% elif render_as == 'yaml' %}
    {{ text_content | render_yaml | pretty }}
```